### PR TITLE
Blog post about the dev exp improvement efforts

### PR DESCRIPTION
**Pull request by hsanjuan**
Add blog post about efforts on the developer experience front.

### CODE REVIEW

File: content/post/091-dev-exp.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* To help funnel users and devs where the community can better support them, our [IPFS discussion forums](https://discuss.ipfs.io/) will be more prominently displayed (in docs, READMEs and issue templates) as the default venue for support (along with pointers to [ipfs.io/help](https://ipfs.io/help)). Some discussion-only and meta-topic repositories are also in the process of migrating their conversations to the forums, where they can reach a wider audience and be more easily searchable and discoverable by other users.
+* New contributors to repositories will now receive automated responses on their first submission, giving context on the process outlined above and reminders for needed information. This will help reduce the number of round trips for contributors and reviewers/maintainers.
+* A [major review of the documentation](https://docs-beta.ipfs.io/) site is already underway to provide better developer documentation for our major implementations and products. Watch this space!
```
Comment (hsanjuan) [inline content/post/091-dev-exp.md:26]: @johnnymatthews can you give me a couple of examples we can link from here?

Comment (johnnymatthews) [reply content/post/091-dev-exp.md:26]: Yeah sure. There's three milestones that will make reader's lives much easier we can mention:

1. [Deprecating the legacy docs site](https://github.com/ipfs/docs/milestone/5).
1. [Single-sourcing documentation](https://github.com/ipfs/docs/milestone/7).
1. [Adding some fancy automation into the docs](https://github.com/ipfs/docs/milestone/6).

File: content/post/091-dev-exp.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+date: 2020-05-05
+url: 2020-05-05-developer-experience
+title: Improving the IPFS Developer experience
+author: Hector Sanjuan
+tags: Developer, Experience
```
Comment (momack2) [inline content/post/091-dev-exp.md:6]: what header image do you want to include?

File: content/post/091-dev-exp.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* To help funnel users and devs where the community can better support them, our [IPFS discussion forums](https://discuss.ipfs.io/) will be more prominently displayed (in docs, READMEs and issue templates) as the default venue for support (along with pointers to [ipfs.io/help](https://ipfs.io/help)). Some discussion-only and meta-topic repositories are also in the process of migrating their conversations to the forums, where they can reach a wider audience and be more easily searchable and discoverable by other users.
+* New contributors to repositories will now receive automated responses on their first submission, giving context on the process outlined above and reminders for needed information. This will help reduce the number of round trips for contributors and reviewers/maintainers.
+* A [major review of the documentation](https://docs-beta.ipfs.io/) site is already underway to provide better developer documentation for our major implementations and products. [Watch this space!](https://github.com/ipfs/docs/milestones)
+
```
Comment (momack2) [inline content/post/091-dev-exp.md:27]: do you want to add an image here of the labelling taxonomy, response bot, or similar?

File: content/post/091-dev-exp.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+* Old issues will not be closed or locked because they are old, or stale, or very low priority. Some parts of the IPFS project have a large backlog and it is important that the information is not lost or dismissed, but can be found by others interested in the topic.
+* The core working groups will still dedicate a large portion of time to help users, answer questions, and review and merge contributions. However, there will be an extra focus on keeping discussions on point and limiting information explosion.
+
+We want to empower the developer community to build amazing things with IPFS, and these contributor experience changes aim to make it faster and easier to start participating in the IPFS Community (whether filing issues or making PRs). However, improving that experience requires feedback and iteration as we refine our next steps as a community. For this, [a new discussion thread](https://discuss.ipfs.io/t/ideas-to-improve-the-ipfs-developer-experience/7750) is available in the forum, where anyone can post new proposals or give suggestions/feedback on recent changes.
+
+We hope these measures help everyone build a happy and growing IPFS developer community!
```
Comment (momack2) [inline content/post/091-dev-exp.md:35]: maybe add the IPFS contributors image here? it's already linked in the 0.5 blog post

File: content/post/091-dev-exp.md (hunk 2 above)
Comment (hsanjuan) [reply content/post/091-dev-exp.md:6]: Can go with defaults or do I need to come up with something custom?

**Comment by hsanjuan** [comment]
@momack2 I have added two images: one shows the auto-responder on a PR with some new labels being used (double-kill), and the contributors thing at the bottom.

**Comment by momack2** [comment]
You can go with the default.

**Pull request merged**